package.json

```diff
@@ -6,17 +6,20 @@
     "start": "nodemon ./src/bin/www"
   },
   "dependencies": {
-    "cookie-parser": "~1.4.4",
-    "debug": "~2.6.9",
-    "express": "~4.16.1",
-    "http-errors": "~1.6.3",
-    "morgan": "~1.9.1",
-    "pg": "^8.4.2",
-    "pug": "2.0.0-beta11",
-    "sequelize": "^6.3.5"
+    "cookie-parser": "~1.4.6",
+    "debug": "~4.3.4",
+    "express": "~4.18.2",
+    "http-errors": "~2.0.0",
+    "morgan": "~1.10.0",
+    "pg": "^8.11.3",
+    "pug": "~3.0.2",
+    "sequelize": "^6.37.0"
+  },
+  "overrides": {
```

Comment:
Can you please remind me again why we are using this override? I tried this out locally without and it is working fine

Reply:
Hi @nbeck415 this was added because one of the packages listed semver as a dependency, but that version has a moderate security vulnerability, so the override allows us to use a more recent version of the semver package. 

Happy to revert it. Wasn't sure how the team handles package versioning or when to update version etc.